doc/api/worker_threads.md

```diff
@@ -1384,7 +1385,7 @@ and what kind of JavaScript values can be successfully transported through
 the thread barrier.
 
 ```mjs
-import assert from 'node:assert';
+import assert from 'node:assert/strict';
```

Comment:
The rest of the changes here are ok but this one is generally unnecessary.

Reply:
[`d7bcad3` (#58264)](https://github.com/nodejs/node/pull/58264/commits/d7bcad3bfcbe18275948dd693d9386474952d93a)